### ISSUE TITLE
[Question] How to enable cast button?

### ISSUE THREAD

**Issue by Amanse** [revanced/revanced-integrations#13]
There is patch to remove cast button but as of latest release it is hidden by default and there's no patch to enable it, is there any way to add it back?

**Comment by oSumAtrIX** [comment]
Disable the patch?

**Comment by Amanse** [comment]
disabling the patch does nothing due to cast button being removed by default in latest release of integration apk

**Comment by oSumAtrIX** [comment]
The patch is what removes it, not the integrations.

**Comment by Canny1913** [comment]
"cast button being removed by default"
uh no it doesnt work like that

we only changed the settings part in integrations, if we dont change it then even if the patch is applied the button would still be there and the patch would be useless. we didn't remove anything at all

are you sure you disabled the patch?

**Comment by oSumAtrIX** [comment]
MicroG relies on the cast button to be disabled.

**Comment by Amanse** [comment]
interesting, but the build i did with the older Integration apk retained the cast button, the newer build removed it

**Comment by oSumAtrIX** [comment]
No, the cast button functionality was disabled anyways due to MicroG. Now it's simply hidden. You can not use casting with microg.

**Comment by Amanse** [comment]
uhh casting works though, i can still cast if i connect TV with code from settings, but having the icon is just easier, avoids the need to go to settings
